plugins/home/backstage-plugin-home-rss/src/RSSCard/Content.tsx

```diff
@@ -60,11 +74,27 @@ export const Content = (props: RSSContentProps) => {
       items.forEach((item) => {
         const link = item.querySelector("link")?.textContent;
         const itemTitle = item.querySelector("title")?.textContent;
+        const pubDate = item.querySelector("pubDate")?.textContent;
+        let pubDateString: string | undefined = undefined;
+        if (pubDate) {
+          const publishedAt = DateTime.fromRFC2822(pubDate);
+          pubDateString = publishedAt.toLocaleString(DateTime.DATE_MED);
+        }
+
 
         if (link && itemTitle) {
+          let itemComponent: any;
+          if (pubDateString) {
+            itemComponent = (<>
+              <Typography className={classes.newsItemDate}>{pubDateString}</Typography>
+              <Link className={classes.newsItemLink} href={link} target="_blank">{itemTitle}</Link>
+            </>);
+          } else {
+            itemComponent = <a href={link} target="_blank">{itemTitle}</a>;
```

Comment:
This should also use `<Link />`. 

Also, I don't think you need to repeat yourself here. Link is always displayed so it doesn't need to be in an `if...else`.

Reply:
done.